torch_geometric/nn/conv/message_passing.py

```diff
@@ -125,11 +126,13 @@ def __init__(
             self.aggr_module = aggr_resolver(aggr, **(aggr_kwargs or {}))
         elif isinstance(aggr, (tuple, list)):
             self.aggr = [str(x) for x in aggr]
-            self.aggr_module = MultiAggregation(aggr, aggr_kwargs)
+            self.aggr_module = MultiAggregation(aggr, aggrs_kwargs=aggr_kwargs,
+                                                **(multi_aggr_kwargs or {}))
         else:
-            raise ValueError(f"Only strings, list, tuples and instances of"
-                             f"`torch_geometric.nn.aggr.Aggregation` are "
-                             f"valid aggregation schemes (got '{type(aggr)}')")
+            raise ValueError(
```

Comment:
What‘s the reason for this diff?

Reply:
Just added a period to keep the style consistent.